mk/declarations.sh

```diff
@@ -161,6 +161,10 @@ WEB_TRUNK_LATEST_BUILD="${WEB_HOST}/carbon/trunk/xe-phase-2-latest"
 WEB_TRUNK_XE_PHASE_1=${WEB_TRUNK_LATEST_BUILD}/xe-phase-1
 TRUNK_GLOBALS=${WEB_TRUNK_XE_PHASE_1}/globals
 
+# REPO_CITRITE_LIB is where repo.citrite.net files can be found
+REPO_CITRITE_HOST="https://repo.citrite.net"
```

Comment:
It would be nice to use this variable in the WEB_HOTFIXES_ROOT as well (line 186)